messages/en.json

```diff
@@ -329,5 +329,26 @@
   "return-app.return-request-details.cancellation.modal.adminAllow": "<p>Attention: This request's status will be set to CANCELLED.</p><p>This will notify the user via email and allow them to create a new return request with the cancelled request's items.</p><p>If that is not your intention, set the request as DENIED.</p>",
   "return-app.return-request-details.cancellation.modal.adminRefuse": "<p>Sorry, it's not possible to cancel this request due to its current status.</p>",
   "return-app.return-request-details.cancellation.modal.storeAllow": "<p>Cancelling this request will allow the current items to be used in a new return request.</p><p>This action is irreversible.</p>",
-  "return-app.return-request-details.cancellation.modal.storeRefuse": "<p>Sorry, you need to contact the support team to cancel this request due to its current status.</p>"
+  "return-app.return-request-details.cancellation.modal.storeRefuse": "<p>Sorry, you need to contact the support team to cancel this request due to its current status.</p>",
+  "admin/return-app.export-module.modal-open": "EXPORT RETURNS",
+  "admin/return-app.export-module.modal-title": "Export module",
+  "admin/return-app.export-module.modal-close": "CLOSE",
+  "admin/return-app.export-module.content.first-paragraph": "The Export module is the system responsible for merging all return requests into a single file called <b>report</b>, this can be then sent to an email or downloaded directly from this module",
+  "admin/return-app.export-module.content.second-paragraph": "Keep in mind that download links are not permanent, they expire after <b>6 hours</b> following their availability",
+  "admin/return-app.export-module.content.warning-paragraph": "It is important to remember that depending on the quantity of return requests, the process can take a long time to finish. If you chose to generate a download link, you can leave the module open and it will update when its available for download",
```

Comment:
Maybe we should rephrase it, since I think the user doesn't need to leave the module open. They can close it and the report will still be running in the background, right?

Reply:
Yeah! It's true that it's difficult to convey the idea of how it works.

The report will run autonomously in the server, but the 'polling' state that continuously updates the frontend with the status ONLY runs when the modal is open. So, there's no way of relying to the user that the export finished.
You've got me thinking, maybe we can shed the context provider outside the modal, this could allow us to couple it with your alert provider and send an alert even if the modal is closed! Exactly like the OMS.

In retrospective I wanted to be stingy on the network calls, so the idea of 'closing' the modal was that you shouldn't have events of something that is not the focus on.

Reply:
With this in mind, what do you think of:

If you chose to generate a download link, you can close the module, or even the page, and come back later; or you can leave it open, and the status on the right will continuously update until its available for download"

Reply:
I wouldn't change the implementation. No need for the pooling call when the modal is not open. I would go about this:

```
The process can take a long time to finish based on the quantity of return requests. Since it runs in the background, you can close this page and check it later. 
```

Reply:
I've changed the text to reflect this